src/NServiceBus.AcceptanceTests/Timeout/When_timeout_storage_is_unavailable_temporarily.cs

```diff
@@ -27,10 +27,10 @@ class When_timeout_storage_is_unavailable_temporarily : NServiceBusAcceptanceTes
         [Test]
         public async Task Endpoint_should_not_shutdown()
         {
-            var stopTime = DateTime.UtcNow.AddSeconds(6);
+            var stopTime = DateTime.UtcNow.AddSeconds(2);
```

Comment:
we've seen parallel tests running for quite a while sometimes, I wonder whether the risk of race conditions between endpoint start and the done condition is to high in this case.

Reply:
We should move this test into the core folder since we're using a `CustomTimeoutPersister` so we should move this to the core folder. Will update the PR